sample-app/manifest.json

```diff
@@ -18,17 +18,28 @@
         }
       },
       "requests": {
-        "iparamValidate":{},
+        "iparamValidate": {},
         "getStarWarsPeople": {},
         "getStarWarsPeopleDetails": {},
         "starwarsStarshipSearch": {},
         "getContacts": {},
-        "replyTicket": {}
+        "replyTicket": {},
+        "dynamicHost": {},
+        "dynamicQueryParams": {},
+        "listAllTickets": {}
+      },
+      "functions": {
+        "usingDynamicHost": {
+          "timeout": 10
+        },
+        "usingQueryParams": {
+          "timeout": 10
+        }
       }
     }
   },
   "engines": {
     "node": "18.13.0",
     "fdk": "9.0.0"
```

Comment:
Could you ensure the right node and fdk version are updated in this app's engine attribute? Because the documentation says, "Ensure that the installed Node version is 18.15.0 or later." Let's validate the same and update the environment accordingly. Also, I wonder if FDK must have received an update to support dynamic host and query params.

Reply:
Node 18.x is good as it's not coupled with sub versions of Node 18. 

FDK version 9.0.5 supports dynamic hosts and query params, it is now available. Let me run it with that version so Manifest is updated accordingly.

Reply:
Updated fdk version to `9.0.5`